pysat/tests/test_utils.py

```diff
@@ -466,77 +463,103 @@ def test_netcdf_attribute_override(self):
         outfile = os.path.join(self.testInst.files.data_path, fname)
         self.testInst.to_netcdf4(outfile)
 
-        data, meta = pysat.utils.load_netcdf4(outfile)
+        _, meta = pysat.utils.load_netcdf4(
+            outfile, pandas_format=self.testInst.pandas_format)
 
         # Custom attribute correctly read from file
         assert meta.bespoke
+        return
+
 
+class TestLoadNetCDF4XArray():
+    """Unit tests for `load_netcdf4` using xarray data.
 
-class TestBasicNetCDF4xarray():
-    """NOTE: combine with above class as part of #60"""
+    Note
+    ----
+    Make this a TestLoadNetCDF4 class test as a part of fixing #60.
+
+    """
 
     def setup(self):
         """Runs before every method to create a clean testing setup."""
-        # store current pysat directory
+        # Store current pysat directory
         self.data_path = pysat.params['data_dirs']
 
-        # create temporary directory
+        # Create temporary directory
         self.tempdir = tempfile.TemporaryDirectory()
         pysat.params['data_dirs'] = [self.tempdir.name]
 
         self.testInst = pysat.Instrument(platform='pysat',
                                          name='testing2d_xarray',
-                                         update_files=True,
-                                         num_samples=100)
+                                         update_files=True, num_samples=100)
         self.stime = pysat.instruments.pysat_testing2d_xarray._test_dates[
             '']['']
 
-        # create testing directory
+        # Create testing directory
         prep_dir(self.testInst)
 
     def teardown(self):
         """Runs after every method to clean up previous testing."""
-        # remove_files(self.testInst)
+        # Reset the pysat parameters
         pysat.params['data_dirs'] = self.data_path
+
+        # Remove the temporary directory
         self.tempdir.cleanup()
-        del self.testInst, self.stime
 
-    def test_basic_write_and_read_netcdf4_default_format(self):
-        """ Test basic netCDF4 writing and reading
```

Comment:
`pysat.utils.load_netcdf4` has different paths depending on pandas or xarray.  Deleting this test without generalizing above (#60) will reduce coverage.